code/shared/KnownPackets.h

```diff
@@ -0,0 +1,59 @@
+#pragma once
+
+#include <utility>
+#include "Utils.h"
+
+constexpr std::pair<uint32_t, const char*> g_knownPackets[]
```

Comment:
`inline` might still be needed in case this ends up used twice in the same module.